### DISCUSSION TITLE
Controller support

### DISCUSSION THREAD

**Discussion by Mister-Lonewolf** [pybricks/support#1507] [Q&A] [answered]
Is it possible to use an "8bitdo sm30 pro+" controller with Pybricks and a four port Lego Technic Powered Up hub, or is it only possible to use an Xbox controller at the time?

**Comment by laurensvalk** [comment, marked as answer]
At this time just the Xbox Controller. It is one of the few that uses Bluetooth low energy that the hub requires.

**Reply by Mister-Lonewolf** [reply]
Ah, thank you for telling me. I didn't know about the low energy part.

**Comment by foux44** [comment]
Hi, is there any news about such Bluetooth controller  ([Bit 8Bitdo Pro SN 30 + ](https://8bitdo.fr/sn30pro-plus.html)) are such remote really not low energy ? Thanks
